code/campaignclassic/src/main/res/layout/push_template_carousel_item.xml

```diff
@@ -1,28 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/carousel_container_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:clipToPadding="false"
     android:clipChildren="false"
+    android:clipToPadding="false"
     android:orientation="vertical"
     android:theme="@style/DayNightTheme">
 
     <ImageView
         android:id="@+id/carousel_item_image_view"
-        android:layout_centerHorizontal="true"
         android:layout_width="250dp"
```

Comment:
nit: we can make the width of carousel item `match_parent` since it needs to occupy the entire available width